### ISSUE TITLE
[TD-6] task/taskd accepts anything as config variable

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskserver#8]
_Renato Alves on 2014-01-22T21:09:48Z says:_ 

 Although this could be considered a feature as it allows for arbitrary values to be added for third party extensions, it also makes debugging hard if a typo gets in by accident and goes unnoticed.

At the moment with taskd there's no way to distinguish recognized from unrecognized settings.
task client provides a different highlight for unrecognized settings.
Ideally taskd would provide a similar distinction mechanism.

As a way to defend from the accidental typo, an additional warning stating that the setting is not recognized could be issued before accepting the value.

Example:

```
task config colour on
Are you sure you want to add 'colour' with a value of 'on'? (yes/no) y
'colour' is not recognized as a valid config variable, do you still wish to accept? (yes/no) y
```

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2014-01-22T21:09:48Z
Modified: 2016-04-09T18:48:02Z
```

**Comment by taskwarrior** [comment]
_Wim Schuermann on 2014-01-22T21:21:42Z says:_ 

 I just realized I lied to you earlier - I did run into problems with this before, but Taskwarrior (the client) does appear to try to push you in the right direction:

```
$ task config asdasdasd 5
Are you sure you want to add 'asdasdasd' with a value of '5'? (yes/no) y
Config file /home/wim/.taskrc modified.
$ task show
(...)
Your .taskrc file contains these unrecognized variables:
    asdasdasd
    
    These are highlighted in color above.
```

I have either never noticed this, or had completely forgotten about it.

This is still a valid request for taskd, which does not support the "show" command. And I really like the "have the user confirm if unknown variables are to be added" idea for both Taskwarrior and taskd.
